<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyMusic</class>
 <widget class="QWidget" name="EasyMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyMusic</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>无歌曲：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="playRateSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="currentTimeLabel">
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="timeSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="totalTimeLabel">
             <property name="text">
              <string>00:00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addSongBtn">
           <property name="text">
            <string>添加歌曲</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showHideListBtn">
           <property name="text">
            <string>显示列表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="playModeComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="preSongBtn">
         <property name="text">
          <string>上一首</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playPauseBtn">
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextSongBtn">
         <property name="text">
          <string>下一首</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="volumeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="volumeLabel">
         <property name="text">
          <string>音量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
